spec/outputs/dynatrace_spec.rb

```diff
@@ -186,6 +186,17 @@
         expect(subject).to have_received(:send_event).exactly(3).times
       end
     end
+
+    context 'with more than 4.5MB of events' do
+      before do
+        allow(subject).to receive(:send_event) { |e, att| [:success, e, att] }
+        subject.multi_receive([*1..2].map { |n| LogStash::Event.new({ 'n' => n.to_s * 2_500_001 }) })
```

Comment:
```suggestion
        subject.multi_receive([1,2].map { |n| LogStash::Event.new({ 'n' => n.to_s * 2_500_001 }) })
```
This would do the same, right?